src/webi-packages/install.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+USERNAME="${USERNAME:-"${_REMOTE_USER:-"automatic"}"}"
```

Comment:
Should username be available as an option in  `src/webi-packages/devcontainer-feature.json`?

Reply:
Okay, makes sense. But why not to omit the `"${USERNAME:-}"` part then?

```bash
USERNAME="${_REMOTE_USER:-"automatic"}"
```

Reply:
Yeah, that's right